### DISCUSSION TITLE
Apps ShowCase on the flutterdata.dev Website.

### DISCUSSION THREAD

**Discussion by drexhacker** [flutterdata/flutter_data#239] [Show and tell]
Hello Everyone I have developing my app and it is heavily backed by `flutter_data` for it's online & offline data fetching and also this packing being completely crossplatform has helped make my app available on all devices. Credit goes to you all especially @frank06 .
I would like my app to also be listed on the `flutterdata.dev` website because it is using flutter_data package at it's core.
You can all feel free to try it out.
<a href='https://play.google.com/store/apps/details?id=app.drexbible.bible&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png' width="200px"/></a>
[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/drexbible)

**Comment by frank06** [comment]
Very cool! Congrats. I will include a link in the README.

**Reply by drexhacker** [reply]
@frank06 Thanks.

**Comment by lewa22suleman** [comment]
wow , like that one))

**Reply by drexhacker** [reply]
@lewa22suleman Thanks too.
